lib/rules/require-numeric-literals.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Requires use of binary, hexadecimal, and octal literals instead of parseInt.
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireNumericLiterals": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * 0b111110111 === 503;
+ * 0o767 === 503;
+ * 0x1F7 === 503;
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * parseInt("111110111", 2) === 503;
+ * parseInt("767", 8) === 503;
+ * parseInt("1F7", 16) === 255;
+ * ```
+ */
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+    configure: function(options) {
+        assert(
+            options === true,
+            this.getOptionName() + ' option requires a true value or should be removed'
+        );
+    },
+
+    getOptionName: function() {
+        return 'requireNumericLiterals';
+    },
+
+    check: function(file, errors) {
+        file.iterateNodesByType(['CallExpression'], function(node) {
+            var callee = node.callee;
+            var args = node.arguments;
+
+            if (args.length !== 2) {
```

Comment:
I guess this is just a return early check since you can have many `CallExpressions` like `a(), a(b), a(b,c,d)` - it's not specifically needed but we do need to know at least 2 exist. Should I just put something like that there?